src/App.tsx

```diff
@@ -54,7 +54,13 @@ injectGlobal`
   }
 
   @media (prefers-color-scheme: dark) {
-    body { background: #333; color: white; }
+    body {
+      background: #333;
+      color: white;
+    }
+    .css-1lybari {
```

Comment:
oh actually, now that I look at this more closely, this class name isn't a guarantee. It's generated during compile time.